composer.json

```diff
@@ -17,26 +17,26 @@
 		"issues": "https://github.com/kdyby/console/issues"
 	},
 	"require": {
-		"php": ">=7.1",
+		"php": ">=7.1 || ^8.0",
 		"nette/di": "^2.4.10 || ^3.0",
 		"nette/routing": "^3.0.0",
 		"tracy/tracy": "^2.5 || ^3.0",
-		"nette/utils": "~3.1.0",
-		"symfony/console": "~2.3 || ^3.0 || < 4.3"
+		"nette/utils": "^2.4 || ^3.0",
+		"symfony/console": "~2.3 || ^3.0 || ^4.0"
 	},
 	"require-dev": {
-		"nette/application": "3.0.*",
+		"nette/application": "^3.0",
 		"nette/bootstrap": "^2.4.5 || ^3.0",
 		"nette/caching": "^2.5 || ^3.0",
 		"nette/http": "^3.0",
-		"kdyby/events": "^3.2@dev",
-		"symfony/event-dispatcher": "~2.3 || ^3.0 || ^4.0 < 4.3",
+		"kdyby/events": "dev-patch-1 as 3.2.99",
+		"symfony/event-dispatcher": "~2.3 || ^3.0 || ^4.0",
 
 		"nette/tester": "^2.2",
-		"phpstan/phpstan-shim": "^0.11.5",
-		"kdyby/coding-standard": "dev-master",
+		"phpstan/phpstan": "0.12.*",
```

Comment:
Lockni to prosím na konkrétní verzi aby další PR nemusel fixovat nové pravidla PHPStanu